### PR TITLE
docs(readme): add webpack notes

### LINKED ISSUE #1656: Build fails out of the box on exif dependency
Building a basic React app out of the box, trying to import `webex` generates the following error:

```js
ERROR in ./node_modules/exif/lib/exif/ExifImage.js
Module not found: Error: Can't resolve 'fs' in '/Users/andrew/Experiments/webex/node_modules/exif/lib/exif'
 @ ./node_modules/exif/lib/exif/ExifImage.js 5:9-22
 @ ./node_modules/exif/lib/exif/index.js
 @ ./node_modules/@webex/helper-image/dist/index.js
 @ ./node_modules/@webex/internal-plugin-conversation/dist/conversation.js
 @ ./node_modules/@webex/internal-plugin-conversation/dist/index.js
 @ ./node_modules/@webex/internal-plugin-calendar/dist/index.js
 @ ./node_modules/webex/dist/webex.js
 @ ./node_modules/webex/dist/index.js
 @ ./src/index.js
```

App is a skeleton React app using webpack and babel to compile:

```js
import React from 'react';
import ReactDOM from 'react-dom';
import { init as webexInit } from 'webex';

const App = () => {
  return (
    <div />
  );
}

ReactDOM.render(<App />, document.getElementById('root'));
```

package.json:

```json
{
  "scripts": {
    "build": "webpack"
  },
  "dependencies": {
    "@babel/core": "^7.9.6",
    "@babel/preset-env": "^7.9.6",
    "@babel/preset-react": "^7.9.4",
    "babel-loader": "^8.1.0",
    "react": "^16.13.1",
    "react-dom": "^16.13.1",
    "webex": "^1.80.199",
    "webpack": "^4.43.0",
    "webpack-cli": "^3.3.11"
  }
}
```

### PR DESCRIPTION
**Pull request by adamweeks**
Resolved #1656

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,7 +181,11 @@ webex.rooms
 4. Run `parcel index.html` in your terminal.
 5. Go to [http://localhost:1234](http://localhost:1234) and open the developer console to see the output.
 
-#### _[Still using `webex/env` or `ciscospark/env`?](documentation/webex.md#shell-script-quick-start)_
+## Node Usage
+
+### `webex/env`
+
+The quickest way to get up and running on node is to utilize the `webex/env` package. See [this documentation](documentation/webex.md#shell-script-quick-start) for guidelines.
```
Comment (InteractiveTimmy) [inline README.md:188]: Looks much clearer!

File: README.md [hunk 2]
```diff
@@ -72,7 +73,7 @@ webex.rooms.create({ title: `My First Room` }).then(room => {
 });
 ```
 
-#### _A note on browser usage_
+### Browser Usage
```
Comment (InteractiveTimmy) [inline README.md:76]: `eslint no-underscore-dangle: "error"`

File: README.md [hunk 3]
```diff
@@ -19,8 +19,9 @@ This is a monorepo containing all officially maintained Cisco Webex JS SDK modul
 - [The Cisco Webex JS SDK](#the-cisco-webex-js-sdk)
   - [Install](#install)
   - [Usage](#usage)
-      - [_A note on browser usage_](#a-note-on-browser-usage)
-      - [_Still using `webex/env` or `ciscospark/env`?_](#still-using-webexenv-or-ciscosparkenv)
+    - [Browser Usage](#browser-usage)
+      - [Browser Usage Example](#browser-usage-example)
+    - [Node Usage](#node-usage)
```
Comment (InteractiveTimmy) [inline README.md:24]: Would be cool to drop a small node usage example.

Comment (adamweeks) [reply README.md:24]: We've got links below it to the quick usage guide in documentation for now.

**Comment by taymoork2** [comment]
Closing in favor of 
https://github.com/webex/webex-js-sdk/pull/1673
https://github.com/webex/webex-js-sdk/pull/1674
https://github.com/webex/webex-js-sdk/pull/1675

**Pull request closed without merging**